src/index.js

```diff
@@ -1,19 +1,36 @@
-import _ from 'lodash';
-import printMe from './print.js';
+/*eslint-disable*/
+import _ from 'lodash'; /*eslint-disable*/
 import './style.css';
 
- function component() {
-   const element = document.createElement('div');
- const btn = document.createElement('button');
+const tasksList = [
+  {
+    description: 'Do homework',
+    completed: false,
+    index: null,
+  },
+  {
+    description: 'Attend Microverse classes',
+    completed: false,
+    index: null,
+  },
+  {
+    description: 'Wash dishes',
+    completed: false,
+    index: null,
+  },
+  {
+    description: 'Do my bed',
+    completed: false,
+    index: null,
+  },
+];
```

Comment:
- Index needs to be a number:
    description [string].
    completed [bool].
    index: [number].